qiskit_ibm_runtime/estimator.py

```diff
@@ -170,8 +170,7 @@ def __init__(
         if session:
             self._session = session
         else:
-            if session_pkg._DEFAULT_SESSION is None:
-                session_pkg._DEFAULT_SESSION = Session(service=service)
+            session_pkg._DEFAULT_SESSION = Session(service=service)
```

Comment:
The idea of `_DEFAULT_SESSION` is to reuse the same session if possible. So this should open a new one _only_ if one doesn't exist or if the existing is closed.